deploy/docker-compose/docFiles/svalinn.yaml

```
@@ -18,17 +18,29 @@ log:
   json: true
 
 endpoint: "/device-status"
-parseQueueSize: 3000
-insertQueueSize: 3000
-maxParseWorkers: 10000
-maxInsertWorkers: 1000
-maxBatchSize: 30
-maxBatchWaitTime: 10ms
-metadataMaxSize: 1000
-payloadMaxSize: 1000
-insertRetries: 0
-retryInterval: 10s
-defaultTTL: 1h
+requestParser:
+  queueSize: 3000
+  maxWorkers: 10000
+  metadataMaxSize: 1000
+  payloadMaxSize: 1000
+  defaultTTL: 1h
+  regexRules:
+    - regex: "device-status/.*/online$"
+      storePayload: true
+      ruleTTL: 1m
+      eventType: "State"
+    - regex: "device-status/.*/offline$"
+      storePayload: true
+      ruleTTL: 1m
```

```suggestion
      ruleTTL: 1m
      eventType: "State"
```
I think state is missing